src/main/java/com/nriker/switter/service/SwitterLikeService.java

```
@@ -0,0 +1,63 @@
+package com.nriker.switter.service;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import com.nriker.switter.model.SwitterLike;
+import com.nriker.switter.model.SwitterPost;
+import com.nriker.switter.model.SwitterUser;
+import com.nriker.switter.persistance.SwitterLikeRepository;
+import com.nriker.switter.persistance.SwitterPostRepository;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+//Todo Service res.
+@Service
+public class SwitterLikeService {
+	@Autowired
+	private SwitterPostService switterPostService;
+	@Autowired
+	private SwitterUserService switterUserService;
+	@Autowired
+    private SwitterLikeRepository likesRepository;
+	
+	public SwitterLike addLike(SwitterLike like) {
+		SwitterUser switterUser = switterUserService.findUserById(like.getUserId());
+		if (switterUser == null) {
+			System.out.println("Der User nicht existiert!");
+			return null;
+		}
+		SwitterPost switterPost = switterPostService.findPostById(like.getPostId());
+		if (switterPost == null) {
+			System.out.println("Das Post nicht existiert!");
```

For the error handling returning "null" is a little unclear, the API should return a 404 in this case - but not it just returns null?
As discussed before, you should throw a Domain Exception, for example PostDoesNotExist extends RuntimeException and then write a @ExceptionHandler in a global @ControllerAdvice for that which then maps the exception to the correct HTTP code.